### DISCUSSION TITLE
Is there a way to share markdown url for specific doc?

### DISCUSSION THREAD

**Discussion by niksfirefly** [toeverything/AFFiNE#6488] [Ideas]
Is there a way to share markdown url for specific doc? or automate exporting doc to markdown file ?

**Comment by pengx17** [comment]
There isn't yet excepting share a doc as it is in the affine's read-only editor. However we do have an external tool to get the markdown output from doc of a cloud workspace using https://github.com/toeverything/affine-reader. Let me know it works for your case.

**Comment by niksfirefly** [comment]
Is there a way to use this tool on self hosted docker with affine cloud workspace?
